certified-connectors/Cloud BOT/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "<client-id>",
+          "scopes": [
+            "execute",
+            "refer",
+            "suspend"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
@CR-mitsubayashi You can keep redirectUrl as "global" if you support multiple redirects per app .